deployments/warp_routes/warpRouteConfigs.yaml

```diff
@@ -5957,6 +5978,108 @@ WETH/artela-base:
       name: Wrapped Ether
       standard: EvmHypCollateral
       symbol: WETH
+WFRAGJTO/fragmetric:
```

Comment:
this is resulting from a `yarn && yarn clean && yarn build`. I think wfragJTO was renamed to WFRAGJTO but for some reason this hasn't been reflected in the built warpRouteConfigs.yaml?

Reply:
the wfragJTO version is the correct one

<img width="144" alt="image" src="https://github.com/user-attachments/assets/4dea75a9-c93e-4eb6-bef5-78e918165792" />